web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,48 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
```

Comment:
Rather than this living in the 4.0 Storage section, what do you think about creating a new .md for "Set Default Storage Class" and linking from the 4.3 section of the README?

Reply:
@lizsurette,  I thought, this particular user action i.e. set default storage class is a small function under the 'Storage' section... so I added to the same 'md' file. 
Are you suggesting to move this particular section (set default storage class) based on the release?

Reply:
@muruGanesan I think you should create a new .md for this feature and put it under the "OpenShift Container Storage" section of the main README.